.github/workflows/analyzers.yaml

```
@@ -3,9 +3,9 @@ permissions: {}
 
 on:
   push:
-    branches: [main]
+    branches: [backport-1.x]
```

I am unsure if it would have any advantage (or disadvantage) to leave the `main` branch as potential target to trigger the workflow.
```suggestion
    branches: [main, backport-1.x]
```

---

I think it would only become more confusing.

We do not intend to keep workflows between branches 100% in sync. So we want the most complete version of the workflow to run on the "main" operations and somewhat reasonable to run on "backport" operations.

To me that means that each branch should have its own workflow file with only desired branch name.

I am also not 100% how github would handle having "main" workflow here. I don't think it would run both workflows as this commit will not belong to main branch, but it all would just confuse me. :smile: